tools/plisttool/plisttool.py

```diff
@@ -1176,6 +1181,15 @@ def _validate_entitlements_against_profile(
                 self.target, aps_environment, profile_aps_environment),
               **report_extras)
 
+    beta_reports_active = entitlements.get('beta-reports-active')
+    if beta_reports_active is not None or profile_entitlements:
```

Comment:
Don't need this `or profile_entitlements`, since the next line handles the case where it's missing.

Reply:
This is for the case if `beta_reports_active is None`